kiro-cli/kiro-cli-config/steering/migration-prompt.md

```diff
@@ -0,0 +1,47 @@
+# Migration Assistant Prompt
+
+## Auto-Discovery
+
+Let me first discover your AWS managed OpenSearch domains across all regions...
+
+```bash
+for r in $(aws ec2 describe-regions --query 'Regions[].RegionName' --output text); do (s=$(aws opensearch list-domain-names --region "$r" --query 'DomainNames[].DomainName' --output text 2>/dev/null) && [ -n "$s" ] && printf "=== %s ===\n%s\n" "$r" "$s") & done; wait
+```
+
+## Choose Migration Type
+
+After discovering your domains, I'll ask:
+
+1. **Migrate between discovered AWS managed domains** (recommended)
+   - I'll show you all discovered domains with their regions
+   - You select source and target from the list
+   - Uses SigV4 authentication automatically
+
+2. **Use custom endpoints** (for self-managed clusters)
+   - Provide source/target endpoints manually
+   - Specify authentication (basic auth or SigV4)
+
+## What I'll Do
+
+1. **Auto-discover domains** - Scan all AWS regions for OpenSearch domains
+2. **Present options** - Show discovered domains or ask for custom endpoints
+3. **Verify connectivity** - Test both clusters are reachable
+4. **Get S3/snapshot config** - From EKS configmap
```

Comment:
should we offer to create the bucket here?  I think that it can be suppressed within the Cfn.